docs/Rest-API.md

```diff
@@ -0,0 +1,147 @@
+# Axe REST API
+
+Documentation of the REST API that comes along with the Axe service.
+
+## AxeResult
+
+This endpoint serves as a wrapper to collect all of the information from an axe run. 
```

Comment:
>endpoint [](start = 5, length = 8)

The docs say "endpoint", then appear to describe the response of the REST API. Shouldn't this document include the information necessary to actually submit the request? Consider, for example, the GitHub API documentation, which include both sample requests and sample responses: [https://developer.github.com/v3/](https://developer.github.com/v3/)